pkg/templates/charts/toggle/console-mce/templates/console-quickstart.yaml

```diff
@@ -0,0 +1,169 @@
+  conclusion: Congratulations! Multicluster switcher is ready to use.
+  durationMinutes: 20
+  icon: >-
+    data:image/svg+xml;base64,PHN2ZyBlbmFibGUtYmFja2dyb3VuZD0ibmV3IDAgMCAxMDAgMTAwIiBoZWlnaHQ9IjEwMCIgdmlld0JveD0iMCAwIDEwMCAxMDAiIHdpZHRoPSIxMDAiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyI+PHBhdGggZD0ibTY2LjcgNTUuOGM2LjYgMCAxNi4xLTEuNCAxNi4xLTkuMiAwLS42IDAtMS4yLS4yLTEuOGwtMy45LTE3Yy0uOS0zLjctMS43LTUuNC04LjMtOC43LTUuMS0yLjYtMTYuMi02LjktMTkuNS02LjktMy4xIDAtNCA0LTcuNiA0LTMuNSAwLTYuMS0yLjktOS40LTIuOS0zLjIgMC01LjIgMi4xLTYuOCA2LjYgMCAwLTQuNCAxMi41LTUgMTQuMy0uMS4zLS4xLjctLjEgMSAuMSA0LjcgMTkuMiAyMC42IDQ0LjcgMjAuNm0xNy4xLTZjLjkgNC4zLjkgNC44LjkgNS4zIDAgNy40LTguMyAxMS40LTE5LjEgMTEuNC0yNC42IDAtNDYuMS0xNC40LTQ2LjEtMjMuOSAwLTEuMy4zLTIuNi44LTMuOS04LjkuNS0yMC4zIDIuMS0yMC4zIDEyLjIgMCAxNi41IDM5LjIgMzYuOSA3MC4yIDM2LjkgMjMuOCAwIDI5LjgtMTAuNyAyOS44LTE5LjIgMC02LjctNS44LTE0LjMtMTYuMi0xOC44IiBmaWxsPSIjZWQxYzI0Ii8+PHBhdGggZD0ibTgzLjggNDkuOGMuOSA0LjMuOSA0LjguOSA1LjMgMCA3LjQtOC4zIDExLjQtMTkuMSAxMS40LTI0LjYgMC00Ni4xLTE0LjQtNDYuMS0yMy45IDAtMS4zLjMtMi42LjgtMy45bDEuOS00LjhjLS4xLjMtLjEuNy0uMSAxIDAgNC44IDE5LjEgMjAuNyA0NC43IDIwLjcgNi42IDAgMTYuMS0xLjQgMTYuMS05LjIgMC0uNiAwLTEuMi0uMi0xLjh6IiBmaWxsPSIjMDEwMTAxIi8+PC9zdmc+
+  introduction: 'With the multicluster switcher, you can easily switch between clusters.'
+  tasks:
+    - title: Enable console plug-in
+      description: |-
+        1. From the Administrator [perspective]{{ "{{" }}highlight qs-perspective-switcher{{ "}}" }} of the OpenShift Container Platform web console, navigate to the cluster settings by clicking
+        **[Administration]{{ "{{" }}highlight qs-nav-administration{{ "}}" }} > Cluster
+        settings**.
+
+        1. Click **Console (operator.openshift.io)** from the *Configuration* tab.
+
+        1. On the *Console plugins* tab, check that the status of the `mce`
+        plug-in is enabled.
+
+        1. If you are also using Red Hat Advanced Cluster Management for Kubernetes, check that
+        the status of the `acm` plug-in is also enabled.
+
+        1. If the applicable plug-ins are disabled, complete the following steps for each plug-in:
+          
+            1. Click on the **Disabled** status.
+         
+            1. Select **Enable** from the dialog box.
+         
+            1. Click on **Save**.  
+
+        1. If you enabled any plug-ins, after a few moments, an alert with the text “Web console update is
+        available” appears. Click the **Refresh web console** link to load the web
+        console with the plug-ins enabled.
+      review:
+        failedTaskHelp: This task is incomplete. Try the task again.
+        instructions: |-
+          Verify that the console plug-in was enabled successfully:
+
+            - In the navigation menu, is a cluster switcher present with the options **All Clusters** and **local-cluster**? The cluster switcher should appear before the [perspective switcher]{{ "{{" }}highlight qs-perspective-switcher{{ "}}" }}.
```

Comment:
Verify that the cluster switcher is present in the navigation menu, with the options **All Clusters** and **local-clusters**. The cluster switcher appears before the [perspective switcher]{{ "{{" }}highlight qs-perspective-switcher{{ "}}" }}.

Make it more of an active tone

Reply:
I provided access to a cluster where you could see the Quick Start in context via the Slack channel: https://coreos.slack.com/archives/C03B532FUNB/p1649959583317009

It's written like this because it gets rendered as a Yes/No question:
<img width="397" alt="image" src="https://user-images.githubusercontent.com/42188127/164555112-06034aa9-3589-498a-b620-4bd9530541d0.png">

Reply:
Thanks for the screenshot! This is helpful. I recommend the following update:

Verify that the console plug-in is enabled successfully.

Reply:
If we change to "is" instead of "was" then I would like to remove the word "successfully". Does that work for you?

Reply:
Updated here: https://github.com/stolostron/backplane-operator/pull/151/commits/bddaee43200acfe78abea025b94b21c7a2a3c724